src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -99,6 +99,11 @@
           "options": {
             "protractorConfig": "./protractor.conf.js",
             "devServerTarget": "ceph-dashboard:serve"
+          },
```

Comment:
I would remove this configuration.
The same result can be achieved with `--dev-server-target` we use in `e2e:dev`

Reply:
@tspmelo whether or not we replace e2e:dev with e2e.ci,
we should reach consensus about what is preferable, having settings defined in
```angular.json``` or having CLI options. It seems that in agunlar project there is tendency to having settings defined in the file.
What should be the recommended approach or 1st option?

Reply:
Not sure if we should use only 1 approach. In my opinion we should use the most simple solution.

p.e.
1. If something can be resolved with 1 flag, I think we should just use it.
2. If we need lots of flags or there are some settings that you can't set via a flag, in that case we should use `angular.json` or an `env`.

Reply:
OK. Done.